README instructions to connect to default MCP Server incorrect

Since #48 none of the documented methods to interact with or connect to the default MCP server seem to work

## Set up
- [WordPress Studio](https://developer.wordpress.com/studio/), with the [REST API authentication workaround for 1.6.0+](https://github.com/Automattic/studio/issues/1878#issuecomment-3455449459). Tested adding a Post via the REST API using Postman to confirm all REST API functionality works as expected.
- [Abilities API trunk](https://github.com/WordPress/abilities-api) cloned, installed and active
- [MCP Adapter trunk](https://github.com/WordPress/mcp-adapter), cloned, installed and active
- [WP Abilities Test](https://github.com/jonathanbossenger/wp-abilities-test) custom plugin installed and active
- WordPress admin application password configured

### WP Abilities Test
The custom plugin checks if the `\WP\MCP\Core\McpAdapter::class` exists, and then creates the default MCP server instance, as [per the readme](https://github.com/WordPress/mcp-adapter?tab=readme-ov-file#using-mcp-adapter-in-your-plugin):

```
// Check if the MCP Adapter plugin is installed and active
if ( class_exists( \WP\MCP\Core\McpAdapter::class ) ) {
    // activate the default server
    \WP\MCP\Core\McpAdapter::instance();
}
```

## Testing MCP Client Configuration in VS Code

###  Test 1: STDIO Transport Configuration for local sites

MCP Server configuration:

```
    "ai-experiments": {
      "command": "wp",
      "args": [
        "--path=/Users/jonathanbossenger/Studio/ai-experiments",
        "mcp-adapter",
        "serve",
        "--server=mcp-adapter-default-server",
        "--user=admin"
      ]
    }
```

VS Code output

```
2025-10-29 11:55:43.870 [info] Starting server ai-experiments
2025-10-29 11:55:43.871 [info] Connection state: Starting
2025-10-29 11:55:43.872 [info] Starting server from LocalProcess extension host
2025-10-29 11:55:43.874 [info] Connection state: Starting
2025-10-29 11:55:43.874 [info] Connection state: Running
2025-10-29 11:55:44.290 [warning] [server stderr] Notice: Function WP_Abilities_Registry::get_instance was called <strong>incorrectly</strong>. Ability API should not be initialized before the <code>init</code> action has fired. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.9.0.) in /Users/jonathanbossenger/Studio/ai-experiments/wp-includes/functions.php on line 6131
2025-10-29 11:55:44.291 [warning] [server stderr] Notice: Function WP_Abilities_Registry::get_instance was called <strong>incorrectly</strong>. Ability API should not be initialized before the <code>init</code> action has fired. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.9.0.) in /Users/jonathanbossenger/Studio/ai-experiments/wp-includes/functions.php on line 6131
2025-10-29 11:55:44.292 [warning] [server stderr] Notice: Function WP_Abilities_Registry::get_instance was called <strong>incorrectly</strong>. Ability API should not be initialized before the <code>init</code> action has fired. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.9.0.) in /Users/jonathanbossenger/Studio/ai-experiments/wp-includes/functions.php on line 6131
2025-10-29 11:55:44.292 [warning] [server stderr] Notice: Function WP_Abilities_Registry::get_instance was called <strong>incorrectly</strong>. Ability API should not be initialized before the <code>init</code> action has fired. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.9.0.) in /Users/jonathanbossenger/Studio/ai-experiments/wp-includes/functions.php on line 6131
2025-10-29 11:55:44.292 [warning] [server stderr] Notice: Function WP_Abilities_Registry::get_instance was called <strong>incorrectly</strong>. Ability API should not be initialized before the <code>init</code> action has fired. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.9.0.) in /Users/jonathanbossenger/Studio/ai-experiments/wp-includes/functions.php on line 6131
2025-10-29 11:55:44.345 [warning] [server stderr] [MCP STDIO Bridge] MCP STDIO Bridge started for server: mcp-adapter-default-server
2025-10-29 11:55:44.348 [info] Discovered 0 tools
```

###  Test 2: HTTP Transport via Proxy

MCP Server configuration:

```
"ai-experiments": {
      "command": "/Users/jonathanbossenger/.nvm/versions/node/v24.9.0/bin/npx",
      "args": ["-y", "@automattic/mcp-wordpress-remote@latest"],
      "env": {
        "PATH": "/Users/jonathanbossenger/.nvm/versions/node/v24.9.0/bin/:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin",
        "WP_API_URL": "https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp",
        "WP_API_USERNAME": "admin",
        "WP_API_PASSWORD": "{LOCAL_APPLICATION_PASSWORD}",
        "LOG_FILE": "/Users/jonathanbossenger/mcp-logs/vscode/ai-experiments-mcp.log"
      }
    }
```

VS Code output

```
025-10-29 11:57:15.270 [info] Stopping server ai-experiments
2025-10-29 11:57:22.072 [info] Starting server ai-experiments
2025-10-29 11:57:22.073 [info] Connection state: Starting
2025-10-29 11:57:22.073 [info] Starting server from LocalProcess extension host
2025-10-29 11:57:22.076 [info] Connection state: Starting
2025-10-29 11:57:22.076 [info] Connection state: Running
2025-10-29 11:57:23.986 [error] Failed to set MCP server log level: Error: MPC -32603: fetch failed
```

ai-experiments-mcp.log contents

```
2025-10-29T09:59:07.090Z [INFO] [SYSTEM] Using native fetch API
2025-10-29T09:59:07.091Z [INFO] [PROXY] Starting WordPress MCP Proxy with enhanced authentication
2025-10-29T09:59:07.093Z [DEBUG] [AUTH] Loaded tokens for server hash: d561e77a2728f38168197d7700fa53a6
2025-10-29T09:59:07.093Z [INFO] [PROXY] MCP server connected to transport successfully
2025-10-29T09:59:07.095Z [INFO] [INIT] 📩 Client Initialize Request received
2025-10-29T09:59:07.095Z [DEBUG] [INIT] Initialize request details:
{
  "method": "initialize",
  "params": {
    "protocolVersion": "2025-06-18",
    "capabilities": {
      "sampling": {},
      "roots": {
        "listChanged": true
      },
      "elicitation": {}
    },
    "clientInfo": {
      "name": "Visual Studio Code",
      "version": "1.105.1"
    }
  }
}
2025-10-29T09:59:07.095Z [INFO] [INIT] 🔄 Initializing WordPress connection with client parameters
2025-10-29T09:59:07.095Z [INFO] [TRANSPORT_DETECT] 🔍 Starting transport detection and WordPress API initialization...
2025-10-29T09:59:07.095Z [INFO] [TRANSPORT_DETECT] 📋 Added proxy information to clientInfo
{
  "originalClient": "Visual Studio Code",
  "proxyName": "@automattic/mcp-wordpress-remote",
  "proxyVersion": "0.2.18"
}
2025-10-29T09:59:07.095Z [INFO] [TRANSPORT_DETECT] 📡 Attempting JSON-RPC transport...
2025-10-29T09:59:07.095Z [DEBUG] [SESSION] MCP JSON-RPC message being sent to WordPress:
{
  "jsonrpc": "2.0",
  "method": "initialize",
  "id": 1,
  "original_mcp_id": "none",
  "session_id": "not-yet-obtained"
}
2025-10-29T09:59:07.095Z [INFO] [TRANSPORT_DETECT] Sending initialization message via JSON-RPC:
{
  "method": "initialize",
  "id": 1,
  "jsonrpc": "2.0",
  "hasParams": true
}
2025-10-29T09:59:07.095Z [DEBUG] [TRANSPORT_DETECT] Complete JSON-RPC init message:
{
  "jsonrpc": "2.0",
  "method": "initialize",
  "id": 1,
  "params": {
    "protocolVersion": "2025-06-18",
    "capabilities": {
      "sampling": {},
      "roots": {
        "listChanged": true
      },
      "elicitation": {}
    },
    "clientInfo": {
      "name": "Visual Studio Code",
      "version": "1.105.1",
      "proxied": {
        "name": "@automattic/mcp-wordpress-remote",
        "version": "0.2.18"
      }
    },
    "_proxy_request_id": 1
  }
}
2025-10-29T09:59:07.095Z [INFO] [API] Request method: initialize (JSON-RPC)
2025-10-29T09:59:07.095Z [DEBUG] [API] JSON-RPC message: {"jsonrpc":"2.0","method":"initialize","id":1,"params":{"protocolVersion":"2025-06-18","capabilities":{"sampling":{},"roots":{"listChanged":true},"elicitation":{}},"clientInfo":{"name":"Visual Studio Code","version":"1.105.1","proxied":{"name":"@automattic/mcp-wordpress-remote","version":"0.2.18"}},"_proxy_request_id":1}}
2025-10-29T09:59:07.095Z [INFO] [AUTH] Using WordPress Basic Auth for method: initialize
2025-10-29T09:59:07.095Z [DEBUG] [AUTH] Username length: 5
2025-10-29T09:59:07.095Z [DEBUG] [AUTH] Password length: 29
2025-10-29T09:59:07.095Z [DEBUG] [AUTH] Auth header length: 48
2025-10-29T09:59:07.095Z [DEBUG] [API] Environment: development
2025-10-29T09:59:07.095Z [DEBUG] [API] Base API URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.096Z [DEBUG] [API] Final requesting URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.096Z [DEBUG] [API] Using Authorization header for authentication
2025-10-29T09:59:07.096Z [INFO] [API] Sending request to WordPress API...
2025-10-29T09:59:07.096Z [DEBUG] [API] Request URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.096Z [DEBUG] [API] Request method: POST
2025-10-29T09:59:07.122Z [ERROR] [API] Error in wpRequest: fetch failed
2025-10-29T09:59:07.122Z [WARN] [TRANSPORT_DETECT] ⚠️  JSON-RPC transport failed, trying simple transport...
2025-10-29T09:59:07.122Z [DEBUG] [TRANSPORT_DETECT] JSON-RPC error details:
{
  "name": "APIError",
  "statusCode": 0,
  "endpoint": "https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp"
}
2025-10-29T09:59:07.122Z [INFO] [TRANSPORT_DETECT] Sending initialization message via Simple transport:
{
  "method": "initialize",
  "hasParams": true
}
2025-10-29T09:59:07.122Z [DEBUG] [TRANSPORT_DETECT] Complete Simple init message:
{
  "method": "initialize",
  "protocolVersion": "2025-06-18",
  "capabilities": {
    "sampling": {},
    "roots": {
      "listChanged": true
    },
    "elicitation": {}
  },
  "clientInfo": {
    "name": "Visual Studio Code",
    "version": "1.105.1",
    "proxied": {
      "name": "@automattic/mcp-wordpress-remote",
      "version": "0.2.18"
    }
  }
}
2025-10-29T09:59:07.122Z [INFO] [API] Request method: initialize (Simple)
2025-10-29T09:59:07.122Z [DEBUG] [API] Simple request: {"method":"initialize","protocolVersion":"2025-06-18","capabilities":{"sampling":{},"roots":{"listChanged":true},"elicitation":{}},"clientInfo":{"name":"Visual Studio Code","version":"1.105.1","proxied":{"name":"@automattic/mcp-wordpress-remote","version":"0.2.18"}}}
2025-10-29T09:59:07.122Z [INFO] [AUTH] Using WordPress Basic Auth for method: initialize
2025-10-29T09:59:07.123Z [DEBUG] [AUTH] Username length: 5
2025-10-29T09:59:07.123Z [DEBUG] [AUTH] Password length: 29
2025-10-29T09:59:07.123Z [DEBUG] [AUTH] Auth header length: 48
2025-10-29T09:59:07.123Z [DEBUG] [API] Environment: development
2025-10-29T09:59:07.123Z [DEBUG] [API] Base API URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.123Z [DEBUG] [API] Final requesting URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.123Z [DEBUG] [API] Using Authorization header for authentication
2025-10-29T09:59:07.123Z [INFO] [API] Sending request to WordPress API...
2025-10-29T09:59:07.123Z [DEBUG] [API] Request URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.123Z [DEBUG] [API] Request method: POST
2025-10-29T09:59:07.128Z [ERROR] [API] Error in wpRequest: fetch failed
2025-10-29T09:59:07.128Z [ERROR] [TRANSPORT_DETECT] ❌ Both JSON-RPC and simple transports failed
{
  "name": "APIError",
  "statusCode": 0,
  "endpoint": "https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp"
}
2025-10-29T09:59:07.128Z [ERROR] [INIT] ❌ Failed to initialize WordPress connection with client parameters
{}
2025-10-29T09:59:07.128Z [WARN] [INIT] ⚠️ Using fallback initialize response
2025-10-29T09:59:07.130Z [INFO] [CLIENT] 📩 Client Message: SetLevel
2025-10-29T09:59:07.130Z [INFO] [CLIENT] Request ID: none | Method: logging/setLevel
2025-10-29T09:59:07.130Z [INFO] [CLIENT] Request parameters:
{
  "paramCount": 1,
  "paramKeys": [
    "level"
  ]
}
2025-10-29T09:59:07.130Z [DEBUG] [CLIENT] Full request parameters:
{
  "level": "info"
}
2025-10-29T09:59:07.130Z [DEBUG] [CLIENT] Session context - ID: none, Transport: detecting, Counter: 1
2025-10-29T09:59:07.130Z [DEBUG] [CLIENT] Complete client request object:
{
  "method": "logging/setLevel",
  "params": {
    "level": "info"
  }
}
2025-10-29T09:59:07.130Z [DEBUG] [MCP] Processing SetLevelRequest
2025-10-29T09:59:07.130Z [INFO] [API] Request method: logging/setLevel (Simple)
2025-10-29T09:59:07.130Z [DEBUG] [API] Simple request: {"method":"logging/setLevel","level":"info"}
2025-10-29T09:59:07.130Z [INFO] [AUTH] Using WordPress Basic Auth for method: logging/setLevel
2025-10-29T09:59:07.131Z [DEBUG] [AUTH] Username length: 5
2025-10-29T09:59:07.131Z [DEBUG] [AUTH] Password length: 29
2025-10-29T09:59:07.131Z [DEBUG] [AUTH] Auth header length: 48
2025-10-29T09:59:07.131Z [DEBUG] [API] Environment: development
2025-10-29T09:59:07.131Z [DEBUG] [API] Base API URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.131Z [DEBUG] [API] Final requesting URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.131Z [DEBUG] [API] Using Authorization header for authentication
2025-10-29T09:59:07.131Z [INFO] [API] Sending request to WordPress API...
2025-10-29T09:59:07.131Z [DEBUG] [API] Request URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.131Z [DEBUG] [API] Request method: POST
2025-10-29T09:59:07.132Z [INFO] [CLIENT] 📩 Client Message: ListTools
2025-10-29T09:59:07.132Z [INFO] [CLIENT] Request ID: none | Method: tools/list
2025-10-29T09:59:07.132Z [INFO] [CLIENT] No request parameters
2025-10-29T09:59:07.132Z [DEBUG] [CLIENT] Session context - ID: none, Transport: detecting, Counter: 1
2025-10-29T09:59:07.132Z [DEBUG] [CLIENT] Complete client request object:
{
  "method": "tools/list",
  "params": {}
}
2025-10-29T09:59:07.132Z [DEBUG] [MCP] Processing ListToolsRequest
2025-10-29T09:59:07.132Z [INFO] [API] Request method: tools/list (Simple)
2025-10-29T09:59:07.132Z [DEBUG] [API] Simple request: {"method":"tools/list"}
2025-10-29T09:59:07.132Z [INFO] [AUTH] Using WordPress Basic Auth for method: tools/list
2025-10-29T09:59:07.132Z [DEBUG] [AUTH] Username length: 5
2025-10-29T09:59:07.132Z [DEBUG] [AUTH] Password length: 29
2025-10-29T09:59:07.132Z [DEBUG] [AUTH] Auth header length: 48
2025-10-29T09:59:07.133Z [DEBUG] [API] Environment: development
2025-10-29T09:59:07.133Z [DEBUG] [API] Base API URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.133Z [DEBUG] [API] Final requesting URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.133Z [DEBUG] [API] Using Authorization header for authentication
2025-10-29T09:59:07.133Z [INFO] [API] Sending request to WordPress API...
2025-10-29T09:59:07.133Z [DEBUG] [API] Request URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.133Z [DEBUG] [API] Request method: POST
2025-10-29T09:59:07.133Z [INFO] [CLIENT] 📩 Client Message: ListPrompts
2025-10-29T09:59:07.133Z [INFO] [CLIENT] Request ID: none | Method: prompts/list
2025-10-29T09:59:07.133Z [INFO] [CLIENT] No request parameters
2025-10-29T09:59:07.133Z [DEBUG] [CLIENT] Session context - ID: none, Transport: detecting, Counter: 1
2025-10-29T09:59:07.133Z [DEBUG] [CLIENT] Complete client request object:
{
  "method": "prompts/list",
  "params": {}
}
2025-10-29T09:59:07.133Z [DEBUG] [MCP] Processing ListPromptsRequest
2025-10-29T09:59:07.133Z [INFO] [API] Request method: prompts/list (Simple)
2025-10-29T09:59:07.133Z [DEBUG] [API] Simple request: {"method":"prompts/list"}
2025-10-29T09:59:07.133Z [INFO] [AUTH] Using WordPress Basic Auth for method: prompts/list
2025-10-29T09:59:07.133Z [DEBUG] [AUTH] Username length: 5
2025-10-29T09:59:07.133Z [DEBUG] [AUTH] Password length: 29
2025-10-29T09:59:07.133Z [DEBUG] [AUTH] Auth header length: 48
2025-10-29T09:59:07.134Z [DEBUG] [API] Environment: development
2025-10-29T09:59:07.134Z [DEBUG] [API] Base API URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.134Z [DEBUG] [API] Final requesting URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.134Z [DEBUG] [API] Using Authorization header for authentication
2025-10-29T09:59:07.134Z [INFO] [API] Sending request to WordPress API...
2025-10-29T09:59:07.134Z [DEBUG] [API] Request URL: https://ai-experiments.wp.local/wp-json/mcp-adapter/v1/mcp
2025-10-29T09:59:07.134Z [DEBUG] [API] Request method: POST
2025-10-29T09:59:07.140Z [ERROR] [API] Error in wpRequest: fetch failed
2025-10-29T09:59:07.140Z [ERROR] [CLIENT] ❌ Client Error: SetLevel failed after 10ms
{
  "error": "fetch failed"
}
2025-10-29T09:59:07.142Z [ERROR] [API] Error in wpRequest: fetch failed
2025-10-29T09:59:07.142Z [ERROR] [CLIENT] ❌ Client Error: ListPrompts failed after 9ms
{
  "error": "fetch failed"
}
2025-10-29T09:59:07.143Z [ERROR] [API] Error in wpRequest: fetch failed
2025-10-29T09:59:07.143Z [ERROR] [CLIENT] ❌ Client Error: ListTools failed after 11ms
{
  "error": "fetch failed"
}
```

--- a/README.md
+++ b/README.md
@@ -417,7 +417,7 @@ Configure MCP clients (Claude Desktop, Claude Code, VS Code, Cursor, etc.) to co
         "@automattic/mcp-wordpress-remote@latest"
       ],
       "env": {
-        "WP_API_URL": "http://your-site.test/wp-json/mcp-adapter/v1/mcp",
+        "WP_API_URL": "http://your-site.test/wp-json/mcp/mcp-adapter-default-server",
         "LOG_FILE": "/path/to/logs/mcp-adapter.log",
         "WP_API_USERNAME": "your-username",
         "WP_API_PASSWORD": "your-application-password"
@@ -461,10 +461,10 @@ add_action('mcp_adapter_init', function($adapter) {
             \WP\MCP\Transport\HttpTransport::class,  // Recommended: MCP 2025-06-18 compliant
         ],
         \WP\MCP\Infrastructure\ErrorHandling\ErrorLogMcpErrorHandler::class, // Error handler
+        \WP\MCP\Infrastructure\Observability\NullMcpObservabilityHandler::class // Observability handler
         ['my-plugin/my-ability'],         // Abilities to expose as tools
         [],                              // Resources (optional)
         [],                              // Prompts (optional)
-        \WP\MCP\Infrastructure\Observability\NullMcpObservabilityHandler::class // Observability handler
     );
 });
 ```

--- a/README.md
+++ b/README.md
@@ -285,7 +285,7 @@ The MCP Adapter automatically creates a default server that exposes all register
 - The default server supports both HTTP and STDIO transports with MCP 2025-06-18 compliance
 - Abilities are exposed as tools, resources, or prompts based on their characteristics
 - Built-in error handling and observability are included
-- Access via HTTP: `/wp-json/mcp-adapter/v1/mcp`
+- Access via HTTP: `/wp-json/mcp/mcp-adapter-default-server`
 - Access via STDIO: `wp mcp-adapter serve --server=mcp-adapter-default-server`
 
 <details>

--- a/README.md
+++ b/README.md
@@ -461,7 +461,7 @@ add_action('mcp_adapter_init', function($adapter) {
             \WP\MCP\Transport\HttpTransport::class,  // Recommended: MCP 2025-06-18 compliant
         ],
         \WP\MCP\Infrastructure\ErrorHandling\ErrorLogMcpErrorHandler::class, // Error handler
-        \WP\MCP\Infrastructure\Observability\NullMcpObservabilityHandler::class // Observability handler
+        \WP\MCP\Infrastructure\Observability\NullMcpObservabilityHandler::class, // Observability handler
         ['my-plugin/my-ability'],         // Abilities to expose as tools
         [],                              // Resources (optional)
         [],                              // Prompts (optional)

--- a/docs/getting-started/README.md
+++ b/docs/getting-started/README.md
@@ -142,7 +142,7 @@ echo '{"jsonrpc":"2.0","id":1,"method":"tools/call","params":{"name":"my-plugin-
 curl "https://yoursite.com/wp-json/"
 
 # Test MCP endpoint (requires authentication)
-curl -X POST "https://yoursite.com/wp-json/mcp-adapter/v1/mcp" \
+curl -X POST "https://yoursite.com/wp-json/mcp/mcp-adapter-default-server" \
   -H "Content-Type: application/json" \
   -d '{"jsonrpc":"2.0","id":1,"method":"tools/list","params":{}}'
 ```

--- a/docs/getting-started/basic-examples.md
+++ b/docs/getting-started/basic-examples.md
@@ -244,7 +244,7 @@ echo '{"jsonrpc":"2.0","id":1,"method":"prompts/list","params":{}}' | wp mcp-ada
 
 ### Default Server
 The MCP Adapter automatically creates a default server that exposes all registered abilities:
-- **Endpoint**: `/wp-json/mcp-adapter/v1/mcp`
+- **Endpoint**: `/wp-json/mcp/mcp-adapter-default-server`
 - **Server ID**: `mcp-adapter-default-server`
 - **Automatic Registration**: All abilities become available immediately
 

--- a/docs/getting-started/basic-examples.md
+++ b/docs/getting-started/basic-examples.md
@@ -106,7 +106,7 @@ add_action( 'wp_abilities_api_init', function() {
 });
 ```
 
-The ability is automatically available via the default MCP server at `/wp-json/mcp-adapter/v1/mcp`.
+The ability is automatically available via the default MCP server at `/wp-json/mcp/mcp-adapter-default-server`.
 
 ### Testing the Tool
 

--- a/docs/getting-started/installation.md
+++ b/docs/getting-started/installation.md
@@ -164,7 +164,7 @@ Alternatively, you can install the MCP Adapter as a traditional WordPress plugin
    wp plugin activate mcp-adapter
    ```
 
-The plugin automatically initializes and creates a default MCP server at `/wp-json/wp-json/mcp/mcp-adapter-default-server`.
+The plugin automatically initializes and creates a default MCP server at `/wp-json/mcp/mcp-adapter-default-server`.
 
 ## Verifying Installation
 

--- a/docs/getting-started/installation.md
+++ b/docs/getting-started/installation.md
@@ -183,7 +183,7 @@ The plugin automatically initializes and creates a default MCP server at `/wp-js
    curl "https://yoursite.com/wp-json/"
    
    # Test MCP endpoint (requires authentication)
-   curl -X POST "https://yoursite.com/wp-json/mcp-adapter/v1/mcp" \
+   curl -X POST "https://yoursite.com/wp-json/mcp/mcp-adapter-default-server" \
      -H "Content-Type: application/json" \
      -d '{"jsonrpc":"2.0","id":1,"method":"tools/list","params":{}}'
    ```

--- a/docs/getting-started/installation.md
+++ b/docs/getting-started/installation.md
@@ -164,7 +164,7 @@ Alternatively, you can install the MCP Adapter as a traditional WordPress plugin
    wp plugin activate mcp-adapter
    ```
 
-The plugin automatically initializes and creates a default MCP server at `/wp-json/mcp-adapter/v1/mcp`.
+The plugin automatically initializes and creates a default MCP server at `/wp-json/wp-json/mcp/mcp-adapter-default-server`.
 
 ## Verifying Installation
 

--- a/docs/guides/transport-permissions.md
+++ b/docs/guides/transport-permissions.md
@@ -16,10 +16,10 @@ $adapter->create_server(
     '1.0.0',
     [\WP\MCP\Transport\HttpTransport::class],
     \WP\MCP\Infrastructure\ErrorHandling\ErrorLogMcpErrorHandler::class,
+    \WP\MCP\Infrastructure\Observability\NullMcpObservabilityHandler::class
     ['my-plugin/tool'], // tools
     [], // resources
     [], // prompts
-    \WP\MCP\Infrastructure\Observability\NullMcpObservabilityHandler::class
     // No permission callback = uses is_user_logged_in()
 );
 ```
@@ -39,10 +39,10 @@ $adapter->create_server(
     '1.0.0',
     [\WP\MCP\Transport\HttpTransport::class],
     \WP\MCP\Infrastructure\ErrorHandling\ErrorLogMcpErrorHandler::class,
+    \WP\MCP\Infrastructure\Observability\NullMcpObservabilityHandler::class,
     ['my-plugin/admin-tool'], // tools
     [], // resources
     [], // prompts
-    \WP\MCP\Infrastructure\Observability\NullMcpObservabilityHandler::class,
     function(): bool {  // Permission callback
         return current_user_can('manage_options');
     }
@@ -157,10 +157,10 @@ $adapter->create_server(
     '1.0.0',
     [\WP\MCP\Transport\HttpTransport::class],
     \WP\MCP\Infrastructure\ErrorHandling\ErrorLogMcpErrorHandler::class,
+    \WP\MCP\Infrastructure\Observability\NullMcpObservabilityHandler::class,
     ['my-plugin/edit-post', 'my-plugin/delete-post'],
     [], // resources
     [], // prompts
-    \WP\MCP\Infrastructure\Observability\NullMcpObservabilityHandler::class,
     function(): bool {
         // Transport: Allow editors and admins
         return current_user_can('edit_posts');

--- a/docs/migration/v0.3.0.md
+++ b/docs/migration/v0.3.0.md
@@ -367,10 +367,10 @@ add_action('mcp_adapter_init', function($adapter) {
         '1.0.0',
         [ HttpTransport::class ],
         ErrorLogMcpErrorHandler::class,
+        NullMcpObservabilityHandler::class,
         ['my-plugin/ability'],
         [],  // resources
         [],  // prompts
-        NullMcpObservabilityHandler::class,
         function() {
             // Custom permission logic
             $api_key = $_SERVER['HTTP_X_API_KEY'] ?? '';

--- a/docs/troubleshooting/common-issues.md
+++ b/docs/troubleshooting/common-issues.md
@@ -172,7 +172,7 @@ add_action( 'rest_api_init', function() {
 ### 401 Unauthorized
 ```bash
 # Test with authentication
-curl -X POST "https://yoursite.com/wp-json/mcp-adapter/v1/mcp" \
+curl -X POST "https://yoursite.com/wp-json/mcp/mcp-adapter-default-server" \
   --user "username:application_password" \
   -H "Content-Type: application/json" \
   -d '{"jsonrpc":"2.0","id":1,"method":"tools/list"}'

